lib/v2/gihyo/group.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const groupId = ctx.params.id;
+
+    const baseUrl = 'https://gihyo.jp';
+    const url = `${baseUrl}/list/group/${groupId}`;
+
+    const response = await got({
+        url,
+        method: 'get',
+    });
```

Comment:
got request can be simplified to 
```suggestion
    const response = await got(url);
```